server/spec/test_mneme_api_response.rb

```diff
@@ -124,9 +131,202 @@ def test_get_mneme_assignment_event
     event = mneme_format.pop
     verify_event event
 
-    assert_equal "1417150740", event[:due_date_sort]
+    assert_equal 1417150740, event[:due_date_sort]
 
     logger.debug "#{__method__}: #{__LINE__}: C: mneme_format: "+mneme_format.inspect
   end
 
+  ################## FILTERING #########################
+  ########### filtering - check published ##############
+  # has been published
+  def test_filter_mneme_assignment_published_and_open
+
+    assign_hash = {
+        'published' => true,
+        'openDate' => 0,
```

Comment:
A non-zero date is a good idea.  A nil openDate means the item is rejected.  I'll make sure the code matches that.